repos/fdbt-site/tests/pages/products/exports.test.tsx

```diff
@@ -6,35 +6,7 @@ describe('pages', () => {
     describe('confirmRegistration', () => {
         it('should render correctly without data', () => {
             const tree = shallow(
-                <Exports
-                    exports={[]}
-                    csrf={''}
-                    myFaresEnabled={true}
-                    exportEnabled={true}
-                    operatorHasProducts={true}
-                />,
-            );
-            expect(tree).toMatchSnapshot();
-        });
-        it('should render correctly with data', () => {
```

Comment:
What's the reason for not testing rendering with data anymore?

Reply:
Because it uses the API getExportProgress to render the data, instead of it being passed to the page in the props.